### DISCUSSION TITLE
go:build comment without error

### DISCUSSION THREAD

**Discussion by joyahamad074** [projectdiscovery/nuclei#1687] [Q&A] [answered]
wheni try to install nuclei current version it shows me 

../go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.6.3/internal/runner/update.go:28:2: //go:build comment without // +build comment
../go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.6.3/cmd/nuclei/main.go:13:2: //go:build comment without // +build comment

this msg and it didn't install

**Comment by ehsandeep** [comment, marked as answer]
@joyahamad074 this is because you are running an outdated version of go, you need **Go 1.17+** to install nuclei
